contracts/SimpleNftMarketplace.sol

```diff
@@ -8,6 +8,11 @@ contract SimpleNftMarketplace is ListingManager, ValidateSignature {
   string private _name;
   string private _version;
 
+  modifier onlyListingOwner(uint256 listingId) {
+    require(msg.sender == _listings[_listingId].seller || isAdmin(msg.sender), 'Only listing owner or moderator');
```

Comment:
Please call the modifier onlyListingOwnerOrModerator  and use the isModerator function

Reply:
Yes, my bad, i took the wrong function